ports/mimxrt10xx/port.mk

```diff
@@ -15,7 +15,11 @@ CFLAGS += \
   -DXIP_EXTERNAL_FLASH=1 \
   -DXIP_BOOT_HEADER_ENABLE=1 \
   -DCFG_TUSB_MCU=OPT_MCU_MIMXRT10XX
-  
+
+# Choose which port to use
+TUD_OPT_RHPORT = 0
```

Comment:
`TUD_OPT_RHPORT` is defined internally by tinyusb, this must be renamed to other such as BOARD_TUD_RHPORT

Reply:
fixed via 9db2504e0443c3b80858c4e6203ef4e05cc2f2de